proto/plan_common.proto

```diff
@@ -60,6 +60,8 @@ message ColumnDesc {
   ColumnDescVersion version = 10;
 
   AdditionalColumn additional_column = 11;
+
+  bool nullable = 12;
```

Comment:
What about https://github.com/risingwavelabs/risingwave/blob/e1c0a93350dc5874c5a2b0fc72a0456c549a4d69/proto/data.proto#L64 🤡

Reply:
🥵 I'll take a look

Reply:
😇 has it used?

Reply:
Seems never used.

Reply:
https://github.com/risingwavelabs/risingwave-legacy/issues/1746

Reply:
Thanks for the information from Xiangjin. As mentioned in [this issue](https://github.com/risingwavelabs/risingwave-legacy/issues/1746#issue-1069133426), if we want to use this field, we would need to change our current simple data types, such as `Boolean`, to more complex types like `Boolean { nullable: bool }`. But `nullable` seems rarely used in our system and it would be quite non-trivial to adapt.

Since we don’t really use it, we should actually consider deprecating it.